### PR TITLE
Fix publishing to Artifactory

### PR DESCRIPTION
**Pull request by arendjr**
# Description

Fixes the authorization for publishing to Artifactory.

# Checklist

Please make sure all of these are checked before merging. Please leave items
you think are non-applicable in the list, but use strike-through (`~~`) to
indicate they don't apply.

- ~~[ ] The changes have been tested to be backwards compatible.~~
- ~~[ ] The OpenAPI schema and generated client have been updated.~~
- ~~[ ] New models module has been added to API generator script~~
- ~~[ ] PR for API is ready and reviewed: (please link)~~
- ~~[ ] PR for Studio is ready and reviewed: (please link)~~
- ~~[ ] PR for CLI is ready and reviewed: (please link)~~
- ~~[ ] PR for FPD is ready and reviewed: (please link)~~
- ~~[ ] The CHANGELOG(s) are updated.~~

When adding new operation types:

- ~~[ ] PR for fiberplane-ot is ready and reviewed: (please link)~~

After merging, please merge related PRs ASAP, so others don't get blocked.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Publish
 
 on:
   push:
-    branches: ["main"]
+    branches: ["main", "fix-artifactory-publishing"]
```
Comment (gagbo) [inline .github/workflows/publish.yml:5]: If you've tested it, you can remove it from the list of branches

**Review by gagbo — APPROVED** [review]
LGTM :)

**Comment by arendjr** [comment]
Thanks! Seems to work so far indeed :) But I'll put it back to draft, because the versions it pushes are not actually correct, so I'll update the Python script first before merging.

**Comment by arendjr** [comment]
I'm closing this one, since the alphas will be published to crates.io as well to keep everything open. But I'll use the fix in here for some of our closed repos :)

**Pull request closed without merging**